src/core/grid_based_algorithms/lb.cpp

```diff
@@ -1556,7 +1576,9 @@ int lb_lbnode_set_pop(int *ind, double *p_pop) {
   return 0;
 }
 
-int lb_lbnode_set_extforce_density(int *ind, double *f) { return -100; }
+int lb_lbnode_set_extforce_density(const Vector3dInt &ind, double *f) {
+  return -100;
```

Comment:
What does `-100` mean?

Reply:
Who knows?

Reply:
I'm not an expert in these matters, but this does not look like a sensible function to me.